ui/component/viewers/videoViewer/internal/plugins/videojs-hls-quality-selector/plugin.js

```diff
@@ -104,11 +106,34 @@ class HlsQualitySelectorPlugin {
 
       concreteButtonInstance.menuButton_.$('.vjs-icon-placeholder').className += icon;
     } else {
-      this.setButtonInnerText('auto');
+      this.setButtonInnerText(QUALITY_OPTIONS.AUTO);
     }
     concreteButtonInstance.removeClass('vjs-hidden');
   }
 
+  resolveAutoQualityLabel(includeResolution) {
+    const player = this.player;
+    const vhs = player.tech(true).vhs;
+
+    if (includeResolution && vhs) {
+      const pixelRatio = this.useDevicePixelRatio ? window.devicePixelRatio || 1 : 1;
+
+      const selectedBandwidth = simpleSelector(
+        vhs.playlists.master,
+        vhs.systemBandwidth,
+        parseInt(safeGetComputedStyle(vhs.tech_.el_, 'width'), 10) * pixelRatio,
+        parseInt(safeGetComputedStyle(vhs.tech_.el_, 'height'), 10) * pixelRatio,
+        vhs.limitRenditionByPlayerDimensions
+      );
+
+      const quality = selectedBandwidth.attributes.RESOLUTION.height;
+
+      return __('Auto (%quality%) --[Video quality popup. Long form.]--', { quality: quality + 'p' });
```

Comment:
Update `app-string` with all edits and additions if possible.

Some tips:
- If it's an edit, check if the existing entry in `app-string` is used elsewhere:
  - If yes, add the string as new entry.
  - If no, edit/replace the existing entry.